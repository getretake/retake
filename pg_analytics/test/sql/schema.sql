CREATE TABLE t (a int, b text NOT NULL) USING deltalake;
INSERT INTO t values (1, 'test');
SELECT * FROM t;
DROP TABLE t;
