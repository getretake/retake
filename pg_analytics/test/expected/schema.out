CREATE TABLE t (a int, b text NOT NULL) USING deltalake;
INSERT INTO t values (1, 'test');
SELECT * FROM t;
 a |  b   
---+------
 1 | test
(1 row)

DROP TABLE t;
